EasyPost.Tests/BatchTest.cs

```diff
@@ -130,10 +131,14 @@ public async Task TestCreateScanForm()
 
 
             Batch batch = await CreateOneCallBuyBatch();
+
+            // Uncomment the following line if you need to re-record the cassette
+            // Thread.Sleep(10000); // Wait enough time to process
+
             await batch.Buy();
 
             // Uncomment the following line if you need to re-record the cassette
-            // Thread.Sleep(2000); // Wait enough time for the batch to process
+            // Thread.Sleep(10000); // Wait enough time to process
```

Comment:
Can we wrap this in an if statement that checks for the existence of the (sadly 4) cassettes in this lib like we have in the others and if any of them are missing to wait 5 seconds? This will remove the need for anyone to dive in to find this line of code and (un)comment it.